uni/lib/view/schedule/schedule.dart

```diff
@@ -124,10 +124,10 @@ class SchedulePageViewState extends GeneralPageViewState<SchedulePageView>
   /// Returns a list of widgets empty with tabs for each day of the week.
   List<Widget> createTabs(MediaQueryData queryData, BuildContext context) {
     final tabs = <Widget>[];
-    final workWeekDays = Provider.of<LocaleNotifier>(context)
-        .getWeekdaysWithLocale()
-        .sublist(0, 5);
-    workWeekDays.asMap().forEach((index, day) {
+    final allDays = Provider.of<LocaleNotifier>(context).getWeekdaysWithLocale()
+      ..removeAt(6);
```

Comment:
What do you think of creating a method in the provider related to the relevant work days (e.g. `getWorkWeekDaysWithLocale()`) and then using it as a private member of this widget class?